### PR TITLE
Fix v44.1 support in instance private api return values

### PR DESCRIPTION
**Pull request by chemikadze**

### CODE REVIEW

File: qubell/api/private/instance.py [hunk 1]
```diff
@@ -119,8 +130,9 @@ def return_values(self):
         #TODO: FIXME: get rid of old API when its support will be removed
         old_api_value = j.get('returnValues')
```
Comment (SergeyKemaev) [inline qubell/api/private/instance.py]: Maybe rename `old_api_value` to 'public_api_value'?

Comment (vasichkin) [inline qubell/api/private/instance.py]: agree

**Comment by vasichkin** [comment]
Looks good.
It would be nice to have unittests for this stuff, if you have time. Could be merged as is.

**Comment by chemikadze** [comment]
Oops, pushed wrong source by mistake, can not force-push new changes. Actual PR with unittests and fixed naming is here https://github.com/qubell/contrib-python-qubell-client/pull/192

**Pull request closed without merging**